pkg/ebpfsyncer/ebpfsyncer.go

```diff
@@ -92,7 +92,7 @@ func (e *ebpfSingleton) SyncInterfaceIngressRules(
 		// Wait for a SIGTERM
 		<-c
 		if e.c != nil {
-			e.resetAll()
+			_ = e.resetAll()
```

Comment:
Should any return value here be logged?

Reply:
not really we are handling `SIGTERM` the container is going away at this point so its best effort cleanup `resetAll` will generate enough logs for error cases